app/components/inplace-boolean.js

```diff
@@ -1,14 +1,19 @@
 import Ember from 'ember';
-import InPlace from 'ilios/mixins/inplace';
+import InPlaceValidation from 'ilios/mixins/inplace-validation';
```

Comment:
In case of `inplace-boolean` component, `inplace-validation` mixin doesn't provide much value other than saving the new value. Should I have `inplace` or `inplace-validation` here? They both work.